src/wp-includes/block-supports/border.php

```diff
@@ -43,6 +40,7 @@ function wp_register_border_support( $block_type ) {
  * attributes array. This will be applied to the block markup in the front-end.
  *
  * @since 5.8.0
+ * @since 6.1.0 Using the style engine to generate CSS and classnames.
```

Comment:
```suggestion
 * @since 6.1.0 Implemented the style engine to generate CSS and classnames.
```
Additional `@since` annotations tend not to use the active tense. [See here](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#since-section-changelogs) for examples.


:1234: Applies elsewhere in the PR.